databases/week3/sql_week3.sql

```diff
@@ -0,0 +1,237 @@
+('Lamb Biryani', 'Indian Lamb Biryani is a casserole of tender lamb curry and fragrant saffron rice. Two outstanding dishes combined into one company worthy casserole. Serve with cooling cucumber raita on the side', 'Aarhus', '2021-07-12 18:30:00', 10, 150, now()),
+('Thai Massaman Lamb Curry', 'Massaman lamb curry is a sumptuous dish that\'s rich in both flavor and nutrients. Here the curry sauce is made from scratch for the best flavor and freshness.', 'Copenhagen', '2021-07-8 12:30:00', 8, 150, now()),
+('Prawn Thai Green Curry', 'The name "green" curry derives from the color of the dish, which comes from green chillies. ... As this is a Thai curry based on coconut milk and fresh green chillies, the color comes out creamy mild green in which the sumptuous praws are cooked.', 'Copenhagen', '2021-07-15 12:30:00', 12, 150, now());
+
+INSERT INTO reservation(number_of_guests, created_date, contact_phonenumber, contact_name, contact_email, meal_id)
+ values (5, now(), '91357857','Redford Dibdall', 'rdibdall0@spotify.com', 1),
+ (6, now(), '89152584','Currey Yegoshin', 'cyegoshin1@phoca.cz', 2),
+ (4, now(), '68726795','Shane Peyto',  'speyto2@chronoengine.com',3),
+ (6, now(), '34328450','Dione Tax',  'dtax3@harvard.edu', 4),
+ (3, now(), '10730873','Dorie Beautyman',  'dbeautyman4@google.ca', 5),
+ (2, now(),'43580083', 'Flore De Gregoli',  'fdegregoli5@edublogs.org',6),
+ (3, now(), '81214295','Kaela Betjes',  'kbetjes6@psu.edu',7),
+ (2, now(), '83530369','Frasquito Trill',  'ftrill7@paypal.com',8),
+ (3, now(), '66338159','Wallis Antoniazzi',  'wantoniazzi8@imdb.com',9),
+ (2, now(), '33777556','Ulberto Brookson',  'ubrookson9@theguardian.com',10),
+ (4, now(), '44734881','Gris de-Ancy Willis',  'gdeancywillisb@nydailynews.com',11);
+ 
+ INSERT INTO review(title, description, meal_id, stars)
+ VALUES ('Excellent food', 'What a lovely homely expereince. It was food for my soul ', 5, 5), 
+ ('Will do it again', 'That was some lovley food: i recommend everone to try it out ', 8, 4), 
+ ('Yummy food', 'Really enjoyed the lamb biryani. I hope they do it agaian and I was lovely talking to the host ', 11, 5), 
+ ('Cold food', 'Did the enjoy the food so much, as it was almost room temperature, and also a bit too spicy for  my pallatte', 10, 3), 
+ ('Delicious Khao Soi', 'It was something I never tried before, but now its one of my favourites. Do try', 8, 5), 
+ ('Simply Wow', 'Me and my friends had a lovely expereince here. Food was simply delish', 6, 5), 
+ ('Lovely vegetarian ', 'It was delicious ', 4, 4);
+ 
+ -- added unique constraint on email as missed out earlier
+  ALTER TABLE reservation 
+ADD CONSTRAINT UNIQUE(contact_email);
+
+-- altered the datatype of price
+ALTER TABLE meal
+MODIFY COLUMN price DECIMAL(5,2);
```

Comment:
Just edit the schema and add it there. You can always start over with `DROP DATABASE meal_sharing;` and running the queries again :)